doc/source/serve/key-concepts.md

```diff
@@ -30,6 +30,20 @@ handle = serve.run(my_first_deployment)
 print(ray.get(handle.remote())) # "Hello world!"
 ```
 
+(serve-key-concepts-application)=
+
+## Application
+
+An application is the unit of upgrade in a Ray Serve cluster. An application consists of one or more deployments. One of these deployments is considered the “ingress” deployment, which is where all inbound traffic is handled.
```

Comment:
ingress deployment should be made a key concept on this page -- it's referenced elsewhere as well

Reply:
it's on the page, a bit further down